NEWS.md

```diff
@@ -1,9 +1,20 @@
+# eesyapi 0.2.0
+
+* Creating publication querying functions:
+  - `get_publication_catalogue()`: Retrieve the list of available publications
+  - `get_publication_datasets()`: Retrieve the list of available data sets in a given publication
+  - `example_id()`: Provide example publication and data set IDs (largely for example code and testing)
+  - `api_url_pages()`: Render string to set paging on API query results
+  - `eesapi_url()` -> `api_url()`: Name change to function and updated to allow publication and data set URLs
+  - `warning_max_pages()`: Check for the query page number exceeding the total query pages available
+  - Added some test data and a process for maintaining that data
```

Comment:
Also added the validate_ family of validation helpers

Reply:
Added